__tests__/fixtures/install/install-should-skip-incompatible-optional-sub-dep/.npmrc

```diff
@@ -0,0 +1 @@
+yarn-offline-mirror=./mirror-for-offline
```

Comment:
No, I don't think it's needed. I was trying to make the changes fit with their surroundings and mimic [this](https://github.com/yarnpkg/yarn/blob/master/__tests__/commands/install/integration.js#L883) test which [uses a tgz file](https://github.com/yarnpkg/yarn/tree/master/__tests__/fixtures/install/install-should-skip-incompatible-optional-dep).

I'm happy to convert the new test to using `file:` dependencies. Should I also convert the one I modeled it after while I'm at it?